### PR TITLE
Folder name updated

### PR DESCRIPTION
**Pull request by Shirish-Panchal-24**
## Summary by CodeRabbit

- **New Features**
	- Introduced a new greeting card layout with enhanced styling for visual appeal.
	- Added comprehensive CSS styles for various greeting card elements, including images and text.
	- Improved layout control for text elements to ensure precise positioning and aesthetics.

- **Bug Fixes**
	- Enhanced overall styling for a better user experience in greeting card presentation.

- **Chores**
	- Updated live server configuration for development environment.

### CODE REVIEW

File: SubmittedAssignments/Shirish_Panchal_12/index.html
Comment (KalpeshApnasite) [inline SubmittedAssignments/Shirish_Panchal_12/index.html:1]: add greeting-background class in index.html

**Review by KalpeshApnasite — CHANGES_REQUESTED** [review]

File: SubmittedAssignments/Shirish_Panchal_12/index.html
Comment (Adarshovhal) [inline SubmittedAssignments/Shirish_Panchal_12/index.html:1]: Add background classes in index. html

**Review by Adarshovhal — CHANGES_REQUESTED** [review]
dont use mm and pt. use only px unit.

**Review by Adarshovhal — CHANGES_REQUESTED** [review]
dont use pt

**Pull request merged**